tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand.cfg

```diff
@@ -28,11 +28,15 @@
 # Copyright (c) 2012 by Delphix. All rights reserved.
 #
 
+#
+# Copyright (c) 2017 Lawrence Livermore National Security, LLC.
+#
+
 
-export org_size=1g
-export exp_size=2g
+export org_size=100m
+export exp_size=200m
```

Comment:
I'm unable to reproduce the error I saw, so I'm dropping those changes to my patch.  When #5679 is merged I'll update to use MINVDEVSIZE.